diff -urNp xfce4-mailwatch-plugin-1.1.0.orig/libmailwatch-core/mailwatch-net-conn.c xfce4-mailwatch-plugin-1.1.0/libmailwatch-core/mailwatch-net-conn.c
--- xfce4-mailwatch-plugin-1.1.0.orig/libmailwatch-core/mailwatch-net-conn.c	2011-09-25 15:10:34.000000000 +0200
+++ xfce4-mailwatch-plugin-1.1.0/libmailwatch-core/mailwatch-net-conn.c	2011-09-25 15:20:28.000000000 +0200
@@ -621,8 +621,10 @@ xfce_mailwatch_net_conn_make_secure(Xfce
                            net_conn->gt_creds);
     gnutls_transport_set_ptr(net_conn->gt_session,
                              (gnutls_transport_ptr_t)net_conn->fd);
+#if GNUTLS_VERSION_NUMBER < 0x020c00
     if(fcntl(net_conn->fd, F_GETFL) & O_NONBLOCK)
         gnutls_transport_set_lowat(net_conn->gt_session, 0);
+#endif
     
     if(!xfce_mailwatch_net_conn_tls_handshake(net_conn, error)) {
 #if 0
