Description: Link format string errors
Author: Yves-Alexis Perez <corsac@debian.org>
Bug: http://bugzilla.xfce.org/show_bug.cgi?id=
Index: xfce4-mailwatch-plugin-1.1.0/libmailwatch-core/mailwatch-net-conn.c
===================================================================
--- xfce4-mailwatch-plugin-1.1.0.orig/libmailwatch-core/mailwatch-net-conn.c
+++ xfce4-mailwatch-plugin-1.1.0/libmailwatch-core/mailwatch-net-conn.c
@@ -224,7 +224,7 @@ xfce_mailwatch_net_conn_tls_handshake(Xf
         else
             reason = gnutls_strerror(ret);
         if(error)
-            g_set_error(error, XFCE_MAILWATCH_ERROR, code, reason);
+            g_set_error(error, XFCE_MAILWATCH_ERROR, code, "%s", reason);
         g_critical("XfceMailwatch: TLS handshake failed: %s", reason);
 
         return FALSE;
@@ -763,7 +763,7 @@ xfce_mailwatch_net_conn_recv_internal(Xf
             break;
         else if(ret < 0 && EINTR != errno) {
             g_set_error(error, XFCE_MAILWATCH_ERROR,
-                        XFCE_MAILWATCH_ERROR_FAILED, strerror(errno));
+                        XFCE_MAILWATCH_ERROR_FAILED, "%s", strerror(errno));
             return -1;
         } else if(!block)
             return 0;
@@ -773,7 +773,7 @@ xfce_mailwatch_net_conn_recv_internal(Xf
     if(ret < 0 && EINTR != errno) {
         if(error) {
             g_set_error(error, XFCE_MAILWATCH_ERROR,
-            XFCE_MAILWATCH_ERROR_FAILED, strerror(errno));
+            XFCE_MAILWATCH_ERROR_FAILED, "%s", strerror(errno));
         }
         return -1;
     } else if(!SHOULD_CONTINUE(net_conn)) {
@@ -785,7 +785,7 @@ xfce_mailwatch_net_conn_recv_internal(Xf
     } else if(TIMER_EXPIRED(RECV_TIMEOUT)) {
         if(error) {
             g_set_error(error, XFCE_MAILWATCH_ERROR,
-                        XFCE_MAILWATCH_ERROR_FAILED, strerror(ETIMEDOUT));
+                        XFCE_MAILWATCH_ERROR_FAILED, "%s", strerror(ETIMEDOUT));
         }
         return -1;
     }
